packages/mdc-list/README.md

```diff
@@ -52,390 +52,132 @@ A basic list consists simply of the list itself, and list items taking up one li
-differentiate them.
+#### List Groups
+Multiple related lists can be grouped together using the mdc-list-group class on a containing element.
 
 ```html
 <div class="mdc-list-group">
   <h3 class="mdc-list-group__subheader">List 1</h3>
   <ul class="mdc-list">
-    <li class="mdc-list-item">Single-line item</li>
-    <li class="mdc-list-item">Single-line item</li>
-    <li class="mdc-list-item">Single-line item</li>
+    <li class="mdc-list-item">line item</li>
+    <li class="mdc-list-item">line item</li>
+    <li class="mdc-list-item">line item</li>
   </ul>
 
-  <hr class="mdc-list-divider">
-
   <h3 class="mdc-list-group__subheader">List 2</h3>
   <ul class="mdc-list">
-    <li class="mdc-list-item">Single-line item</li>
-    <li class="mdc-list-item">Single-line item</li>
-    <li class="mdc-list-item">Single-line item</li>
+    <li class="mdc-list-item">line item</li>
+    <li class="mdc-list-item">line item</li>
+    <li class="mdc-list-item">line item</li>
   </ul>
 </div>
 ```
 
-### Tips/Tricks
-
-#### Bordered Lists
-
-While hinted at within the spec, **bordered lists** - where each list item has a border around
-it - are not officially part of the list component spec. However, they seem to be used
-often in web applications, especially those suited more for desktop. The following example shows how
-to add borders to lists.
+#### List Dividers
+MDC List contains an mdc-list-divider class which can be used as full-width or inset subdivisions either within lists themselves, or standalone between related groups of content.
 
 ```html
-<style>
-  .my-bordered-list {
-    /* remove the side padding. we'll be placing it around the item instead. */
-    padding-right: 0;
-    padding-left: 0;
-  }
-  .my-bordered-list .mdc-list-item {
-    /* Add the list side padding padding to the list item. */
-    padding: 0 16px;
-    /* Add a border around each element. */
-    border: 1px solid rgba(0, 0, 0, .12);
-  }
-  /* Ensure adjacent borders don't collide with one another. */
-  .my-bordered-list .mdc-list-item:not(:first-child) {
-    border-top: none;
-  }
-</style>
-<!-- ... -->
-<ul class="mdc-list my-bordered-list">
-  <li class="mdc-list-item">Item 1</li>
-  <li class="mdc-list-item">Item 2</li>
-  <li class="mdc-list-item">Item 3</li>
+<ul class="mdc-list">
+  <li class="mdc-list-item">Item 1 - Division 1</li>
+  <li class="mdc-list-item">Item 2 - Division 1</li>
+  <li role="separator" class="mdc-list-divider"></li>
+  <li class="mdc-list-item">Item 1 - Division 2</li>
+  <li class="mdc-list-item">Item 2 - Division 2</li>
 </ul>
 ```
 
-#### Control Tile Positions
+> Note the role="separator" attribute on the list divider. It is important to include this so that assistive technology can be made aware that this is a presentational element and is not meant to be included as an item in a list. Note that separator is indeed a valid role for li elements.
```

Comment:
nit: Style the note here similar to notes in the [Text Field readme](https://github.com/material-components/material-components-web/tree/master/packages/mdc-textfield) e.g.
> _NOTE_: add note here